test-clients/src/main/java/com/hedera/services/bdd/suites/contract/ContractCallSuite.java

```diff
@@ -706,27 +709,53 @@ HapiApiSpec payableSuccess() {
 										resultWith().logs(
 												inOrder(
 														logWith().longAtBytes(depositAmount, 24)))))
-														);
+				);
 	}
 
-	HapiApiSpec callingContract() {
```

Comment:
Did you mean to delete the `callingContract` test?  Seems relevant.

Reply:
I have moved it to the `CallOperationSuite`